cime_config/namelist_definition_drv.xml

```diff
@@ -2109,43 +2109,47 @@
     <type>char</type>
     <category>aux_hist</category>
     <group>MED_attributes</group>
-    <desc>Auxiliary mediator wav2med file1 output fields (colon delimited)</desc>
+    <desc>Auxiliary mediator wav2med average history output every day.</desc>
     <values>
-      <value>Sw_hs:Sw_wlm:Sw_thm:Sw_thp0:Sw_fp0:Sw_u:Sw_v:Sw_ustokes:Sw_vstokes:Sw_tusx:Sw_tusy</value>
+      <value>Sw_hs:Sw_Tm1:Sw_thm:Sw_thp0:Sw_fp0:Sw_u:Sw_v:Sw_ustokes:Sw_vstokes:Sw_tusx:Sw_tusy</value>
     </values>
+    <desc>Auxiliary mediator wav2med file1 output fields (colon delimited)</desc>
   </entry>
   <entry id="histaux_wav2med_file1_history_option">
     <type>char</type>
     <category>aux_hist</category>
     <group>MED_attributes</group>
-    <desc>Auxiliary mediator wav2med file1 output option</desc>
+    <desc>history option type</desc>
     <values>
       <value>ndays</value>
     </values>
+    <desc>Auxiliary mediator wav2med file1 output option</desc>
   </entry>
   <entry id="histaux_wav2med_file1_history_n">
     <type>integer</type>
     <category>aux_hist</category>
     <group>MED_attributes</group>
-    <desc>Auxiliary mediator wav2med file1 output frequency (used for option type)</desc>
+    <desc>history option type</desc>
     <values>
       <value>1</value>
     </values>
+    <desc>Auxiliary mediator wav2med file1 output frequency (used for option type)</desc>
   </entry>
   <entry id="histaux_wav2med_file1_doavg">
     <type>logical</type>
     <category>aux_hist</category>
     <group>MED_attributes</group>
-    <desc>Auxiliary mediator wav2med file1 time averaged flag for file output.</desc> 
+    <desc>If true, use time average for aux file output.</desc>
```

Comment:
Should this be:
```suggestion
    <desc>If .true., use time average for aux file output.</desc>
```
Also, what happens for `.false.`?

Reply:
@gold2718 - I have updated the comments.